### ISSUE TITLE
Installing on django > 1.7?

### ISSUE THREAD

**Issue by anyong** [django-guardian/django-guardian#301]
I am running the release candidate for 1.8, and I can't apply a migration after setting guardian in my INSTALLED_APPS. Keep getting this error:

There is no South database module 'south.db.postgresql_psycopg2' for your database. Please either choose a supported database, check for SOUTH_DATABASE_ADAPTER[S] settings, or remove South from INSTALLED_APPS.

It seems like all of the migrations in the guardian/migrations modules are still using South? But the docs say django-guardian is compatible with 1.7... what do?

Thanks.

**Comment by troygrosfield** [comment]
I ran across this issue and I simply uninstalled south since I wasn't using it and the error message went away.  

If you actually need south, uninstall and reinstall and see if that works. [Others have had success](http://stackoverflow.com/questions/6172209/how-should-i-be-setting-south-database-adapters-for-my-django-app-that-uses-sout) with that for fixing this error (not quite sure why that works).
